src/test/java/io/nats/client/support/JwtUtilsTests.java

```diff
@@ -48,7 +48,7 @@ public void issueUserJWTSuccessMinimal() throws Exception {
 
         String jwt = issueUserJWT(SIGNING_KEY, ACCOUNT_ID, new String(USER_KEY.getPublicKey()), null, null, null, 1633043378, "audience");
         String claimBody = getClaimBody(jwt);
-        String cred = String.format(NATS_USER_JWT_FORMAT, jwt, new String(USER_KEY.getSeed()));
+        String cred = String.format(io.nats.jwt.JwtUtils.NATS_USER_JWT_FORMAT, jwt, new String(USER_KEY.getSeed()));
```

Comment:
Could instead have one import of `io.nats.jwt.JwtUtils.NATS_USER_JWT_FORMAT` on top. But just a small thing.

Reply:
This needs to complete test the deprecated version of this field, so I did remove this fully qualified part, but so the field tested is the one in the deprecated class. I checked all the imports too.